docs/tutorials/initializedirtyimage.py

```diff
@@ -0,0 +1,190 @@
+#
+#
+
+# ### Image Setup
+#
+# Here we will be setting up the ALMA Logo image dataset and displaying it. Consult the Gridding and Diagnostic Images Tutorial for reference.
+
+import torch
+import numpy as np
+import matplotlib.pyplot as plt
+from mpol import gridding, coordinates, precomposed, losses, utils
+from astropy.utils.data import download_file
+from torch.utils.tensorboard import SummaryWriter
+
+# In this tutorial we will be saving and loading the model we use and it is important to note that when saving and loading a model the cell_size, nchan, and npix are the same between the model and the coordinates if used in another document.
+
+# +
+# load the mock dataset of the ALMA logo
+fname = download_file(
+    "https://zenodo.org/record/4498439/files/logo_cube.npz",
+    cache=True,
+    show_progress=True,
+    pkgname="mpol",
+)
+
+# this is a multi-channel dataset... but for demonstration purposes we'll use
+# only the central, single channel
+chan = 4
+d = np.load(fname)
+uu = d["uu"][chan]
+vv = d["vv"][chan]
+weight = d["weight"][chan]
+data_re = d["data_re"][chan]
+data_im = -d["data_im"][
+    chan
+]  # we're converting from CASA convention to regular TMS convention by complex conjugating the visibilities
+
+# define the image dimensions, making sure they are big enough to fit all
+# of the expected emission
+coords = coordinates.GridCoords(
+    cell_size=0.03, npix=180
+)  # The smaller the cell size and the larger the npix can greatly increase run time
+gridder = gridding.Gridder(
+    coords=coords,
+    uu=uu,
+    vv=vv,
+    weight=weight,
+    data_re=data_re,
+    data_im=data_im,
+)
+
+# export to PyTorch dataset
+dset = gridder.to_pytorch_dataset()
+# -
+
+# Show the dirty image
+img, beam = gridder.get_dirty_image(weighting="briggs", robust=0.0, unit="Jy/beam")
+kw = {"origin": "lower", "extent": gridder.coords.img_ext}
+fig, ax = plt.subplots(ncols=1)
+snp = ax.imshow(np.squeeze(img), **kw)
+ax.set_title("image")
+ax.set_xlabel(r"$\Delta \alpha \cos \delta$ [${}^{\prime\prime}$]")
+ax.set_ylabel(r"$\Delta \delta$ [${}^{\prime\prime}$]")
+plt.colorbar(snp)
+
+# ### Model and Optimization Setup
+#
+# Here we will be setting up the optimizer and the image model (RML). If this is unfamiliar please reference the Optimization tutorial. The initial parameters of the model are also displayed and can be contrasted with them after the optimization loop. }
```

Comment:
I think it would be nice to include a link to the optimization tutorial here. Also, this may be a bit nitpicky, but I think you should consider rephrasing the tutorial to use more direct language, e.g. "Here we will be setting up the optimizer" --> "Here we set up the optimizer"